app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -118,9 +119,11 @@ class Settings private constructor(
         default = true
     )
 
+
+    // If autoPlayMedia is flagged OFF, default to true here
     val isAutoPlayEnabled = getSitePermissionsPhoneFeatureAction(
         PhoneFeature.AUTOPLAY, Action.BLOCKED
-    ) != Action.BLOCKED
+    ) != Action.BLOCKED || !FeatureFlags.autoPlayMedia
```

Comment:
Normally, I'd flip the short-circuit to avoid wasting cycles, but this will work.